idaes/power_generation/flowsheets/sofc/surrogates/cpu.py

```diff
@@ -7,8 +7,20 @@
 # Washington to design NGFC systems with high efficiencies and low CO2
 # emissions.
 ##############################################################################
+##############################################################################
```

Comment:
Is this file intended in this PR?

Reply:
I had to fix those files to pass the tests